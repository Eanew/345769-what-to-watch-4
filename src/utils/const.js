export const Screen = {
  MAIN: `main`,
  DETAILS: `details`,
};

export const APPROVED_GENRES = [
  `Comedy`,
  `Crime`,
  `Documentary`,
  `Drama`,
  `Horror`,
  `Kids & Family`,
  `Romance`,
  `Sci-Fi`,
  `Thriller`];

export const Rating = {
  'Awesome': 10,
  'Very good': 8,
  'Good': 5,
  'Normal': 3,
  'Bad': 0,
};
