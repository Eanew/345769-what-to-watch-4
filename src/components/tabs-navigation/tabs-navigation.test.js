import React from "react";
import renderer from "react-test-renderer";
import TabsNavigation from "./tabs-navigation.jsx";

import {Tab} from "../../utils/const.js";

it(`Should TabsNavigationComponent render correctly`, () => {
  const tree = renderer.create(
      <TabsNavigation
        currentTab={Tab.REVIEWS}
        onTabClick={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
