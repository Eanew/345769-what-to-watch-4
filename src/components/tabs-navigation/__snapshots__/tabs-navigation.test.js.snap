// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TabsNavigationComponent render Reviews 1`] = `
Array [
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="movie-card__reviews movie-card__row"
  >
    <div
      className="movie-card__reviews-col"
    >
      <div
        className="review"
      >
        <blockquote
          className="review__quote"
        >
          <p
            className="review__text"
          >
            Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
          </p>
          <footer
            className="review__details"
          >
            <cite
              className="review__author"
            >
              Viktor Savinov
            </cite>
            <time
              className="review__date"
              dateTime="2019-05-08"
            >
              May 08, 2019
            </time>
          </footer>
        </blockquote>
        <div
          className="review__rating"
        >
          3,2
        </div>
      </div>
      <div
        className="review"
      >
        <blockquote
          className="review__quote"
        >
          <p
            className="review__text"
          >
            Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
          </p>
          <footer
            className="review__details"
          >
            <cite
              className="review__author"
            >
              Alexandra Pushina
            </cite>
            <time
              className="review__date"
              dateTime="2019-05-08"
            >
              May 08, 2019
            </time>
          </footer>
        </blockquote>
        <div
          className="review__rating"
        >
          4,2
        </div>
      </div>
      <div
        className="review"
      >
        <blockquote
          className="review__quote"
        >
          <p
            className="review__text"
          >
            Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
          </p>
          <footer
            className="review__details"
          >
            <cite
              className="review__author"
            >
              Julia Kim
            </cite>
            <time
              className="review__date"
              dateTime="2019-05-08"
            >
              May 08, 2019
            </time>
          </footer>
        </blockquote>
        <div
          className="review__rating"
        >
          5,2
        </div>
      </div>
    </div>
    <div
      className="movie-card__reviews-col"
    >
      <div
        className="review"
      >
        <blockquote
          className="review__quote"
        >
          <p
            className="review__text"
          >
            Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
          </p>
          <footer
            className="review__details"
          >
            <cite
              className="review__author"
            >
              Anton Timoshenkov
            </cite>
            <time
              className="review__date"
              dateTime="2019-05-08"
            >
              May 08, 2019
            </time>
          </footer>
        </blockquote>
        <div
          className="review__rating"
        >
          6,2
        </div>
      </div>
    </div>
  </div>,
]
`;
